scripts/build-merge

```diff
@@ -127,6 +127,11 @@ echo-do cp examples/movies/spark/target/scala-2.10/movies-spark.jar ${STAGE_MOVI
 echo-do mkdir -p stage/docs/scala/store
 echo-do mv doc/target/scala-2.11/unidoc stage/docs/scala/store/${version}
 
+#set NSEEDS
```

Comment:
This comment tells me what the line of code tells me.  Maybe change the comment to:
Number of seeds for psuedo-random tests.